src/common/marketUtils.es6

```diff
@@ -1,15 +1,43 @@
+const market_order = {
+    forex: 1,
+    indices: 2,
+    commodities: 3,
+    volidx: 4,
+};
+
+const getMarketPosition = (markets) => markets.sort((a, b) => market_order[a] - market_order[b]);
```

Comment:
Since `market_order` is only used in `getMarketPosition `, maybe we can hide it with a closure within `getMarketPosition `?